pkg/controller/directimagemigration/directimagemigration_controller.go

```diff
@@ -117,17 +118,20 @@ func (r *ReconcileDirectImageMigration) Reconcile(request reconcile.Request) (re
 		if errors.IsNotFound(err) {
 			// Object not found, return.  Created objects are automatically garbage collected.
 			// For additional cleanup logic use finalizers.
-			return reconcile.Result{}, nil
+			return reconcile.Result{Requeue: false}, nil
 		}
 		// Error reading the object - requeue the request.
-		return reconcile.Result{}, err
+		return reconcile.Result{Requeue: true}, err
 	}
 
 	// Completed.
 	if imageMigration.Status.Phase == Completed {
-		return reconcile.Result{}, nil
+		return reconcile.Result{Requeue: false}, nil
 	}
 
+	// Default re-queue (after) in seconds.
+	requeueAfter := time.Duration(PollReQ)
```

Comment:
PollReQ is the default, and it is overridden by the output of `r.migrate` below. This is the combination that has been proven to work well, tested heavily during removal of remote watch. 

After removing the TouchAnnotation (which causes constant echo reconciles), I saw migrations halt in the middle because they weren't using the output of r.Migrate in DIM, DISM, DVM, DVMP.

Reply:
+1. thanks for digging into this, makes a lot of sense

Reply:
for locality, do you think moving this to just before the if block at line 145 makes sense? would make the code much more readable when coming back to this in the future.

Reply:
Yeah I think that makes sense, will change 👍